src/spikeinterface/comparison/comparisontools.py

```diff
@@ -232,7 +176,61 @@ def compute_matching_matrix(
     return compute_matching_matrix
 
 
-def make_match_count_matrix(sorting1, sorting2, delta_frames):
+def make_match_count_matrix(sorting1, sorting2, delta_frames, symmetric=False):
+    """
+    Compute a matrix representing the matches between two Sorting objects.
+
+    Given two spike trains, this function finds matching spikes based on a temporal proximity criterion
+    defined by `delta_frames`. The resulting matrix indicates the number of matches between units
+    in `spike_frames_train1` and `spike_frames_train2` for each pair of units.
+
+    Note that this algo is not symmetric and biased toward sorting1 is the ground truth.
+
+    Parameters
+    ----------
+    sorting1 : Sorting
+        An array of integer frame numbers corresponding to spike times for the first train. Must be in ascending order.
+    sorting2 : Sorting
+        An array of integer frame numbers corresponding to spike times for the second train. Must be in ascending order.
+    delta_frames : int
+        The inclusive upper limit on the frame difference for which two spikes are considered matching. That is
+        if `abs(spike_frames_train1[i] - spike_frames_train2[j]) <= delta_frames` then the spikes at `spike_frames_train1[i]`
+        and `spike_frames_train2[j]` are considered matching.
+    symmetric: bool, dfault False
```

Comment:
I wish we could come up with a better name of this but I don't have anything better.

Reply:
Like, it makes me expected the returning matrix to be symmetric but that can't be.

Reply:
`force_symmetry` ? `ensure_symmetry` ? sounds better ?

Reply:
Both are good choices to me actually. The two very good.